guides/v9-hammerjs-migration.md

```diff
@@ -91,25 +91,25 @@ from the module. This is automatically done by the migration.
 
 There are other patterns where the deprecated `GestureConfig` is extended, injected or used
 in combination with a different custom gesture config. These patterns cannot be handled
-automatically, but the migration will report such patterns and ask for manual cleanup.
+automatically, but the migration will report such patterns and ask you to perform manual cleanup.
 
 <a name="test-migration"></a>
 ### How to migrate my tests?
 
 Components in your project might use Angular Material components which previously depended
-on HammerJS. There might be unit tests for these components which also test gesture functionality
-of the Angular Material components. For such unit tests, find all failing gesture tests. These
-might need to be reworked to dispatch proper events to simulate gestures, or need to be deleted.
-Specifically gesture tests for the `<mat-slide-toggle>` should be removed. This is because the
-`<mat-slide-toggle>` no longer supports gestures.
+upon HammerJS. There might be unit tests for these components which also test gesture functionality
+of the Angular Material components. For such unit tests, you can find all failing gesture tests.
+Then you can rework these tests to dispatch proper events, in order to simulate gestures, or
```

Comment:
They _might_ need to rework these tests. Why has this been changed? To me, it now sounds like it's optional. I personally think we should be more instructive here since we are in the `How to migrate my tests?` section.

Reply:
Isn't the choice that they can rework the tests or remove them?

It sounds like you are saying that there are 3 choices?
1. do nothing
1. rework the tests
1. remove the tests

Reply:
There are actually two options, and it always depends on the situation (as grouped by the scenarios).

1. Tests for Angular Material components that used gestures
   * Remove the unit tests completely

2. Tests for custom components that rely on the gesture infrastructure brought in by Material
   * Rework the tests
   * Remove the tests completely

I don't think there is a third option (doing nothing; unless there are no failing tests for sure).

Reply:
This code block (were the comment sits on) is about the (1) scenario where Angular Material components are tested, and tests need to be either removed or reworked (depending on the app).

I think your proposal is better, but I'd just change it from `can` to `should` because right now it implies that there is also the option of doing nothing (because both times `can` is used).

Reply:
OK, we're on the same page then.

My proposal is that we say that "you can rework the tests..., or the tests can be deleted". This is probably passive voice, so I'll update it to "you can rework the tests..., or delete the tests."